.env

```diff
@@ -2,3 +2,4 @@
 PORT=3000
 REACT_APP_USERNAME=$USER
 REACT_APP_ENV="development"
+REACT_APP_VIP="http://vip.localhost:3001"
```

Comment:
could we name this like `REACT_APP_VIP_HOST` or something that offers more meaning?